### ISSUE TITLE
local variable 'annotation_data' referenced before assignment 

### ISSUE THREAD

**Issue by davidpickup** [yhenon/keras-frcnn#91]
When running `python train_frcnn.py `, I get the following output:
```
Parsing annotation files                                                                                                               │
local variable 'annotation_data' referenced before assignment
local variable 'annotation_data' referenced before assignment
Segmentation fault (core dumped)
```

**Comment by Ahleroy** [comment]
Hello,

As explained into the ReadMe file, you have to launch the script using : 
`python train_frcnn.py -p ./pathToLabelsOfDataSet/`

If you want to use the simple parser then use : 
`python train_frcnn.py -o simple -p ./pathToLabelsOfDataSet/`

Regards,

**Comment by davidpickup** [comment]
Sorry to confuse you, I did pass this argument, I just didn't include it above sorry :S

I've also had to change some code in the parser, as the structure of the VOC dataset linked to in the README is different from the structure assumed in the code. The differences were that there is no VOC2007 folder, and it also assumed that there are a `trainval.txt` and `test.txt` split in the `Main` folder within `ImageSets`, when there is a `train.txt`, `trainval.txt` and a `val.txt`, but no `test.txt`. I've corrected these things in the code, but I still get the above error.
